### PR TITLE
Updates to the Readme

### PR DESCRIPTION
**Pull request by tanmayeekamath**
Added the following:

- Section on usage with a default kitchen sink image 
- Updates on how to forward a port

### CODE REVIEW

File: script-library/docs/sshd.md [hunk 1]
```diff
@@ -21,7 +21,34 @@
 
 ## Usage
 
-This script can be used either ad-hoc in an already running container or in a Dockerfile.
+This script can be used either ad-hoc in an already running container or in a Dockerfile. 
+
+### Usage with a default container image
```
Comment (Chuxel) [inline script-library/docs/sshd.md:26]: This repository is also used for Remote - Containers repository where no concept of a default image exists.  

What I'd recommend is we say, "Usage when this script is already in an image (e.g. the default Codespaces image - codespaces-linux)" or something like that.

**Review by Chuxel — CHANGES_REQUESTED** [review]

File: script-library/docs/sshd.md (hunk 1 above)
Comment (tanmayeekamath) [reply script-library/docs/sshd.md:26]: Got it.. that makes sense. I'll update the title and resubmit!

**Comment by tanmayeekamath** [comment]
I have made the suggested updates now.  Thanks for the feedback!

**Comment by Chuxel** [comment]
LGTM!

**Pull request merged**